exampleCourse/questions/element/imageCapture/question.html

```diff
@@ -0,0 +1,23 @@
+<pl-question-panel>
+<p>This question demonstrates the image capture element, which enables students to submit images to questions using their local camera or an external device (e.g. a phone or tablet).</p>
```

Comment:
Nit for consistent indentation (sorry we don't have tooling to handle this yet!)

```suggestion
  <p>This question demonstrates the image capture element, which enables students to submit images to questions using their local camera or an external device (e.g. a phone or tablet).</p>
```